src/database/schema/UserSchema.ts

```diff
@@ -359,7 +374,9 @@
       anon_user_info_color_code: object?.anon_user_info_color_code,
       anon_user_info_color_name: object?.anon_user_info_color_name,
       anon_user_info_emoji_code: object?.anon_user_info_emoji_code,
-      anon_user_info_emoji_name: object?.anon_user_info_emoji_name
+      anon_user_info_emoji_name: object?.anon_user_info_emoji_name,
+      isAnonymous: object?.is_anonymous ? IS_ANONYMOUS : IS_NOT_ANONYMOUS
+      // TODO: add is_anonymous
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Please remove the TODO comment if the task has been completed.

```diff
-        // TODO: add is_anonymous
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->